ClassNotFoundException: StatusAggregator for Spring Boot < 2.2

Resilience4j version: 1.7.0

Java version: 8

Using `resilience4j-spring-boot2` with spring boot version < 2.2 and actuator results in this exception:
```
Caused by: java.lang.ClassNotFoundException: org.springframework.boot.actuate.health.StatusAggregator
```
due to `CircuitBreakersHealthIndicatorAutoConfiguration` and `RateLimitersHealthIndicatorAutoConfiguration` classes being loaded, which depend on `StatusAggregator` class which was added in Spring boot 2.2.0. This is fixable by simply adding this class to the `ConditionalOnClass` annotation, and we are currently using resilience4j packaged with just that change with spring boot version 2.0.x in our projects. I'll submit a PR for the same.

--- a/resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/autoconfigure/CircuitBreakersHealthIndicatorAutoConfiguration.java
+++ b/resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/autoconfigure/CircuitBreakersHealthIndicatorAutoConfiguration.java
@@ -16,7 +16,7 @@ import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ConditionalOnClass({CircuitBreaker.class, HealthIndicator.class})
+@ConditionalOnClass({CircuitBreaker.class, HealthIndicator.class, StatusAggregator.class})
 @AutoConfigureAfter(CircuitBreakerAutoConfiguration.class)
 @AutoConfigureBefore(HealthContributorAutoConfiguration.class)
 public class CircuitBreakersHealthIndicatorAutoConfiguration {

--- a/resilience4j-spring-boot2/src/main/java/io/github/resilience4j/ratelimiter/autoconfigure/RateLimitersHealthIndicatorAutoConfiguration.java
+++ b/resilience4j-spring-boot2/src/main/java/io/github/resilience4j/ratelimiter/autoconfigure/RateLimitersHealthIndicatorAutoConfiguration.java
@@ -16,7 +16,7 @@ import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ConditionalOnClass({RateLimiter.class, HealthIndicator.class})
+@ConditionalOnClass({RateLimiter.class, HealthIndicator.class, StatusAggregator.class})
 @AutoConfigureAfter(RateLimiterAutoConfiguration.class)
 @AutoConfigureBefore(HealthContributorAutoConfiguration.class)
 public class RateLimitersHealthIndicatorAutoConfiguration {

